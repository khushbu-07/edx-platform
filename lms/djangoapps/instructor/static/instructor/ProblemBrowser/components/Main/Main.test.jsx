import React from 'react';
import Main from './Main.jsx';
import renderer from 'react-test-renderer';

test('My first demo test', () => {
  const component = renderer.create(<Main></Main>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
