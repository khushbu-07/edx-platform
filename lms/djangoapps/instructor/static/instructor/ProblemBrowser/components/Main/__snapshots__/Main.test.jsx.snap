// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first demo test 1`] = `
<div
  className="problem-browser"
>
  <button
    className="btn"
    onBlur={[Function]}
    onClick={[Function]}
    onKeyDown={[Function]}
    type="button"
  >
    Select a section or problem
  </button>
  <input
    disabled={true}
    name="problem-location"
    type="text"
  />
</div>
`;
